builds/e2e/templates/longhaul-deploy-windows.yaml

```diff
@@ -0,0 +1,39 @@
+parameters:
+  release.Label: ''
+  edgelet.artifact.name: ''
+  images.artifact.name: ''
+  container.registry: ''
+  container.registry.username: ''
+  container.registry.password: ''
+  iotHub.connectionString: ''
+  eventHub.connectionString: ''
+  snitch.build.number: ''
+  snitch.alert.url: ''
+  snitch.storage.account: ''
+  snitch.storage.masterKey: ''
+  loadGen.message.frequency: ''
+
+steps:
+  - task: CopyFiles@2
+    displayName: 'Copy Edgelet Artifact'
+    inputs:
+      SourceFolder: "$(System.ArtifactsDirectory)/${{ parameters['edgelet.artifact.name'] }}"
+      TargetFolder: "$(Agent.HomeDirectory)/../artifacts/${{ parameters['edgelet.artifact.name'] }}"
+      CleanTargetFolder: true
+  - task: CopyFiles@2
+    displayName: 'Copy Images Artifact'
+    inputs:
+      SourceFolder: "$(System.ArtifactsDirectory)/${{ parameters['images.artifact.name'] }}"
+      TargetFolder: "$(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}"
+      CleanTargetFolder: true
+  - task: PowerShell@2
+    displayName: 'Run Long Haul Deployment'
+    inputs:
+      targetType: inline
+      script: |
+        #Requires -RunAsAdministrator
+        $testName="LongHaul"
+        $ArtifactInfo=convertfrom-stringdata (get-content $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/artifactInfo.txt -raw)
+        Invoke-Expression $(Agent.HomeDirectory)/../artifacts/${{ parameters['images.artifact.name'] }}/scripts/windows/setup/IotEdgeSecurityDaemon.ps1
```

Comment:
why do you need this line? Run-E2ETest.ps1 should call iotedgequickstart.exe, which will call IoTEdgeSecurityDaemon.ps1 to install iot edge.

Reply:
Function Cleanup is called before iotedgequickstart.exe as part of the setup. Uninstall-IoTEdge is called in there and would fail without calling IoTEdgeSecurityDaemon.ps1. I can move that to inside the script since I thought it was called somewhere else for the e2e tests, but then realized it is part of iotedgequickstart.exe.

Reply:
Move this inside runE2Etests.ps1 will be better. :)